File: 1047 - Game Time with Minutes/Main.js
let [ih, im, fh, fm] = require("fs").readFileSync("/dev/stdin", "utf8").split(" ").map((n) => parseInt(n));
let tm = (fh - ih) * 60 + fm - im;
if (tm < 1) {
  tm += 1440;
}
console.log(`O JOGO DUROU ${(tm / 60) | 0} HORA(S) E ${tm % 60} MINUTO(S)`);

File: 1047 - Game Time with Minutes/Main.py
ih, im, fh, fm = map(int, input().split())
tm = (fh - ih) * 60 + fm - im
if tm < 1:
    tm += 1440
print(f'O JOGO DUROU {tm // 60} HORA(S) E {tm % 60} MINUTO(S)')
